pkg/snapshotcombiner/statscombiner_test.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snapshotcombiner
+
+import "testing"
+
+func TestStatCombiner(t *testing.T) {
+	sc := NewSnapshotCombiner[int](2)
+
+	type testRun struct {
+		Name            string
+		IntervalStats   map[string][]int
+		ExpectedEntries int
+	}
+
+	tests := []testRun{
```

Comment:
I was a bit confused at the beginning because I was thinking each of these was a different test, but those are "steps" of the same test that should run one after the other. Do you think there is a way to make it more clear?